receiver/telemetrygeneratorreceiver/go.mod

```diff
@@ -0,0 +1,42 @@
+module github.com/observiq/bindplane-agent/receiver/telemetrygeneratorreceiver
+
+go 1.21.6
```

Comment:
Can we add this go mod to the dependabot file. Example for MS365 receiver.

https://github.com/observIQ/bindplane-agent/blob/release/v1.46.0/.github/dependabot.yml#L155-L165